NextJS project on scaffold always errors out

**Describe the bug**


(node:internal/process/task_queues:95:5) at async fetchLatestStableVersion2 (/Users/user/.nvm/versions/node/v20.18.3/lib/node_modules/corepack/dist/lib/corepack.cjs:21672:14) at async Engine.getDefaultVersion (/Users/user/.nvm/versions/node/v20.18.3/lib/node_modules/corepack/dist/lib/corepack.cjs:22298:23) at async Engine.executePackageManagerRequest (/Users/user/.nvm/versions/node/v20.18.3/lib/node_modules/corepack/dist/lib/corepack.cjs:22396:47) at async Object.runMain (/Users/user/.nvm/versions/node/v20.18.3/lib/node_modules/corepack/dist/lib/corepack.cjs:23102:5) Node.js v20.18.3
npm error code ERESOLVE
npm error ERESOLVE unable to resolve dependency tree npm error npm error While resolving: next-template@0.1.0 npm error Found: react@19.1.0 npm error node_modules/react npm error react@"^19.0.0" from the root project npm error npm error Could not resolve dependency: npm error peer react@"^16.8.0 || ^17.0.0 || ^18.0.0" from react-day-picker@8.10.1 npm error node_modules/react-day-picker npm error react-day-picker@"^8.10.1" from the root project npm error npm error Fix the upstream dependency conflict, or retry npm error this command with --force or --legacy-peer-deps npm error to accept an incorrect (and potentially broken) dependency resolution. npm error npm error npm error For a full report see: npm error /Users/user/.npm/_logs/2025-06-02T21_00_16_601Z-eresolve-report.txt
npm error A complete log of this run can be found in: /Users/user/.npm/_logs/2025-06-02T21_00_16_601Z-debug-0.log
**To Reproduce**
Steps to reproduce the behavior:

1. Create any nextjs and ask it to generate anything
2. I need to go to cursor to fix issue
3. Scroll down to '....'
4. See error

**Desktop (please complete the following information):**

- OS: iOS
- Browser chrome
- Version all

--- a/src/ipc/handlers/app_handlers.ts
+++ b/src/ipc/handlers/app_handlers.ts
@@ -69,7 +69,7 @@ async function executeAppLocalNode({
   event: Electron.IpcMainInvokeEvent;
 }): Promise<void> {
   const process = spawn(
-    "(pnpm install --force && pnpm run dev --port 32100) || (npm install --legacy-peer-deps && npm run dev -- --port 32100)",
+    "(pnpm install && pnpm run dev --port 32100) || (npm install --legacy-peer-deps && npm run dev -- --port 32100)",
     [],
     {
       cwd: appPath,

--- a/src/ipc/handlers/app_handlers.ts
+++ b/src/ipc/handlers/app_handlers.ts
@@ -69,7 +69,7 @@ async function executeAppLocalNode({
   event: Electron.IpcMainInvokeEvent;
 }): Promise<void> {
   const process = spawn(
-    "(pnpm install && pnpm run dev --port 32100) || (npm install && npm run dev -- --port 32100)",
+    "(pnpm install --force && pnpm run dev --port 32100) || (npm install --legacy-peer-deps && npm run dev -- --port 32100)",
     [],
     {
       cwd: appPath,

--- a/src/ipc/processors/executeAddDependency.ts
+++ b/src/ipc/processors/executeAddDependency.ts
@@ -19,7 +19,7 @@ export async function executeAddDependency({
   const packageStr = packages.join(" ");
 
   const { stdout, stderr } = await execPromise(
-    `(pnpm add ${packageStr}) || (npm install ${packageStr})`,
+    `(pnpm add ${packageStr}) || (npm install --legacy-peer-deps ${packageStr})`,
     {
       cwd: appPath,
     },

